plugins/SitesManager/API.php

```diff
@@ -1155,23 +1155,25 @@ public function setGlobalExcludedReferrers(string $excludedReferrers): void
 
         $excludedUrls = $this->checkAndReturnCommaSeparatedStringList($excludedReferrers);
 
-        foreach (explode(',', $excludedUrls) ?: [] as $url) {
```

Comment:
I guess the `?: []` check was already meant to catch empty values, but was implemented incorrectly.
Simply using this here, should already fix everything:
```
foreach (!empty($excludedUrls) ? explode(',', $excludedUrls) : [] as $url) {
```